socorro/siglists/irrelevant_signature_re.txt

```diff
@@ -8,6 +8,7 @@
 __aeabi_fcmpgt.*
 ashmem
 app_process@0x.*
+AppleIntelHD3000GraphicsGLDriver@.*
```

Comment:
These frames are super long and were causing some of the crashes I was looking at to get truncated, so I added them to this list to get ignored.